src/workerd/api/hibernatable-web-socket.c++

```diff
@@ -106,6 +105,25 @@ kj::Promise<WorkerInterface::CustomEvent::Result> HibernatableWebSocketCustomEve
         KJ_UNREACHABLE;
       }
     });
+
+  try {
+    KJ_IF_SOME(hm, manager) {
```

Comment:
Could this be `auto& hm = KJ_REQUIRE_NONNULL(manager);` instead? We should be failing and complaining if we don't have one but with a `KJ_IF_SOME` we just drop the promise.